mvpa2/mappers/procrustean.py

```diff
@@ -208,3 +208,15 @@ def _train(self, source):
             d_r = np.linalg.norm(odatas[0] - res_r)/np.linalg.norm(odatas[0])
             debug('MAP_', "%s, residuals are forward: %g,"
                   " reverse: %g" % (repr(self), d_f, d_r))
+
+
+    def _compute_recon(self):
+        """For Procrustean mapper, inverse is transpose.
+        So, let's skip computing inverse in the super class.
+        """
+        # XXX Change pinv to superclass compute_recon?
+        if self.params.oblique:
+            #return ProjectionMapper._compute_recon(self)
+            return np.linalg.pinv(self._proj)
+        else:
+            return np.transpose(self._proj/self._scale**2) if self.params.scaling else np.transpose(self._proj)
```

Comment:
Well, to reverse map from target to source, we need to scale the target by `1/scale` for the same reason we multiply forward mapping matrix by scale in the main function. Does it make sense?
If not, run
r=get_random_rotation(2,2)*0.5
rpinv=np.linalg.pinv(r)
rtscaled=r.T/0.25
np.allclose(rpinv, rtscaled)